components/automate-cs-nginx/habitat/config/chef_https.conf

```diff
@@ -246,8 +246,8 @@
     # HABITAT: NO ADDONS YET
     #    include /var/opt/opscode/nginx/etc/addon.d/*_external.conf;
 
-    # oc-id
-    include {{pkg.svc_config_path}}/oc-id-external.conf;
```

Comment:
how come OCID was already there in this component?

Reply:
Possibly this got added when the automate team was trying to add chef-server as automate components.